app/components/collection_metadata_component.rb

```diff
@@ -16,7 +16,6 @@ class CollectionMetadataComponent < ActionView::Component::Base
     :title,
     :subtitle,
     :creator_aliases,
-    :description,
```

Comment:
@rschenk why did description get removed?

Reply:
@awead Description is no longer shown in the metadata listing in the new UI design, it's shown in its own paragraph in the main content area of the view